### DISCUSSION TITLE
Consider documenting `opts.indent.indent` somehwere

### DISCUSSION THREAD

**Discussion by nadiamoe** [folke/snacks.nvim#333] [Ideas]
Took me a good chunk of time to figure out why my indent configuration wasn't working, and only https://github.com/folke/snacks.nvim/discussions/314 gave me the clue.

This is not correct:
```lua
{
  "folke/snacks.nvim",
  opts = {
    indent = {
      only_scope = true,
  },
},
```

This is:
```lua
{
  "folke/snacks.nvim",
  opts = {
    indent = {
      indent = {
        only_scope = true,
      },
    },
  },
},
```

As many things this is obvious in hindsight, seeing that the docs specify `indent` and e.g. `animate` at the same level, but nesting `indent` inside of `indent` felt strange enough that my brain completely discarded that option.

**Comment by cyrus303** [comment]
thank you, i was struggling to setup this config for indent

**Comment by folke** [comment]
It is documented. Not my fault that people can't read.

**Reply by nadiamoe** [reply]
Redundancy is good. If people struggle to find the place where this information is stated (which it is), it is helpful to note it somewhere else in a slightly different form (e.g. a comment on the examples).

**Comment by windowsrefund** [comment]
It's just a poorly named namespace.
